IntegrationAdapters/Models/Medicine.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace IntegrationAdapters.Models
+{
+    public class Medicine
+    {
+        public String MedicineID { get; set; }
+        public String Name { get; set; }
+        public String MedicineSpecificationID { get; set; }
+
+        public Medicine()
+        {
+        }
+        public Medicine(string medicineID, string name, String medicineSpecification)
```

Comment:
rename medicineSpecification to medicineSpecificationId as it fits better

Reply:
and you should always use string instead of String - the difference is kind of like int and Integer in Java (one is a simple type and the other is a class)